main.go

```diff
@@ -78,12 +99,12 @@ func (m *MozProcessStat) Add(data MozProcessStat) {
 	m.Memory.Swap += data.Memory.Swap
 	// Available memory must not be summed.
```

Comment:
Nit: I think we can remove this comment as `Available` is no longer part of the original `ProcMemoryInfoStat`  struct anyway.